examples/managementcli/main.go

```diff
@@ -328,15 +328,15 @@ func permissionAll(args []string) error {
 }
 
 func roleCreate(args []string) error {
-	return descopeClient.Management.Role().Create(context.Background(), args[0], flags.Description, flags.Permissions)
+	return descopeClient.Management.Role().Create(context.Background(), args[0], flags.Description, flags.Permissions, "")
```

Comment:
Why not to get it from args[1]

Reply:
Because you will need to create a tenant first
This is just a sample app
Don't think it is an important part of the sample

Reply:
Developers can use this CLI after the tenant already exists